docs/variants/msi_z690/gpu-hcl.md

```diff
@@ -32,5 +32,8 @@ Information about GPU might be read from GPU package or documentation.
 |:----------------:|:-----------:|:-----------:|:---------:|:------------------:|:-------------------------:|:----------------------:|
 | Nvidia GeForce GTX 1060 | 3072 MB | GDDR5 | 192GB/s      | Gen3               | 1                            | [Qubes HCL reports][1] |
 | MSI Radeon RX 6950 XT | 16GB | GDDR6 | 576GB/s           | Gen4               | 1                            |  |
+| EVGA NVidia RTX 2080 | 8GB | GDDR6 | 448GB/s           | Gen3               | 1                            |  |
+| PNY NVidia RTX A5000 | 24GB | GDDR6 | 768GB/s           | Gen4               | 1                            |  |
```

Comment:
@Keno were you able to enter the firmware Setup menu with these GPU cards?

Reply:
I didn't try entering the menu, but on both cards the Dasharo boot splash that told me what buttons to press to enter the menu showed up. I can boot the machine again trying to actually enter the menu.

Reply:
If the boot splash and button information appeared when display was plugged to Nvidia card, it is good enough. Thank you.